### PR TITLE
Add recipe for GraphAligner

### PR DESCRIPTION
**Pull request by maickrau**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

@bioconda/core I would like a review as this is my first recipe and the pull request adds a new package.

### CODE REVIEW

File: recipes/graphaligner/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  version: 1.0.0
+
+source:
+  url: https://github.com/maickrau/GraphAligner/files/2764381/GraphAligner.tar.gz
+  sha256: 6bbebd3c7373c2018245391c8ecdc97cc58e2afa82d3523e08afd0407df1bf63
+  patches: version.patch
+
+build:
+  number: 0
+  skip: False # [linux]
```
Comment (bgruening) [inline recipes/graphaligner/meta.yaml]: you can remove this line

File: recipes/graphaligner/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    - {{ compiler('cxx') }}
+  host:
+    - jemalloc
+    - zlib
+    - mummer4
+    - boost
+    - libboost
+    - sparsehash
+    - pkg-config
+    - libdivsufsort
```
Comment (bgruening) [inline recipes/graphaligner/meta.yaml]: Do you need to pin this? Is the library API stable?

File: recipes/graphaligner/meta.yaml (hunk 1 above)
Comment (maickrau) [reply recipes/graphaligner/meta.yaml]: Thanks, removed

File: recipes/graphaligner/meta.yaml (hunk 2 above)
Comment (maickrau) [reply recipes/graphaligner/meta.yaml]: Turns out that libdivsufsort was getting statically linked even though the makefile suggests that it should be dynamically linked. I removed it from the run: section and it still seems to work.

**Pull request merged**